### ISSUE TITLE
Sleep on restart

### ISSUE THREAD

**Issue by iuriguilherme** [joar/mediagoblin-init-scripts#9]
You should add `sleep 1` between the stop and start on the restart command as done on [this fork](https://notabug.org/piratas/mediagoblin-init-scripts/commit/99f66dc3c28362b5fc42906e20e78c0601c2f00b).

It is a workaround for the fact that the celery-worker process sometimes take too long to stop and when the new process tries to start, the pid file is already there. There are other ways to solve this issue, but since a link to this repository is on the [deploy recipe](http://mediagoblin.readthedocs.org/en/stable/siteadmin/production-deployments.html), you should at least patch it.

Actually the script should handle this the "proper" way as the Debian guidelines says, but whatever.

**Comment by Rovanion** [comment]
Yeah, it should be patched. But only for legacy reasons since the future lies in systemd unit files now that both Debian and Ubuntu, along with the RedHat world since long, has moved to it.
